docs/software/apps-and-envs/gaussian.md

```diff
@@ -0,0 +1,141 @@
+
+## Example Gaussian job with `sinteractive`
+You can start an `sinteractive` session on Midway3 by running this command in your terminal. Don't forget to replace `pi-drpepper` with your account (typically your PI's CNetID).
+
+```
+$ sinteractive --partition=caslake --account=pi-drpepper --nodes=1 --ntasks-per-node=8 --mem-per-cpu=16000 --time=00:30:00
+```
+
+Next, load Gaussian by running this command in your terminal. Substitute `16RevA.03` with whichever version of Gaussian you would like to use. (Remember you can see available versions of Gaussian by running `module avail gaussian` in your terminal.)
+
+```
+$ module load gaussian/16RevA.03
+```
+
+Now you can run Gaussian with the `g#` command. Match `#` to whatever version of Gaussian you have loaded: `g16` for `16RevA.03`, `g09` for `gaussian09`, etc. For example, this code uses `16RevA.03` to run a file called `your-input.com` and write the results to `your-output.out`:
+
+```
+$ g16 < your-input.com > your-output.out
+```
+
+If you need to make more complex specifications, use an `sbatch` file (see below) to submit your job to the RCC's [Slurm workload manager](../../slurm/main.md), which will run the job on a compute node for you.
+
+## Example Gaussian job with `sbatch`
+
+Here is an example input file, `water.com`:
+
+```
+%mem=16GB
+%nprocshared=8
+# HF/6-31G(d)
+
+water energy
+
+0   1
+O  -0.464   0.177   0.0
+H  -0.464   1.137   0.0
+H   0.441  -0.143   0.0
+```
+
+Here is an example `sbatch` file, `water.sbatch`, that runs `water.com`:
+
+```
+#!/bin/bash
+#SBATCH --job-name=example_sbatch_gaussian
+#SBATCH --partition=caslake
+#SBATCH --time=00:30:00
+#SBATCH --account=rcc-staff # replace this with your PI account 
+#SBATCH --nodes=1
+#SBATCH --ntasks-per-node=1
+#SBATCH --cpus-per-task=8
+#SBATCH --mem=16000 # in Megabytes
+#SBATCH --error=example.err 
+
+module unload gaussian
+module load gaussian/16RevA.03
+
+g16 < water.com > $SLURM_JOB_ID.out
+
+touch $SLURM_JOB_ID.txt 
+echo "Job ID: $SLURM_JOB_ID" >> $SLURM_JOB_ID.txt 
+echo "Job name: $SLURM_JOB_NAME" >> $SLURM_JOB_ID.txt
+echo "N tasks: $SLURM_ARRAY_TASK_COUNT" >> $SLURM_JOB_ID.txt
+echo "N cores: $SLURM_CPUS_ON_NODE" >> $SLURM_JOB_ID.txt
+echo "N threads per core: $SLURM_THREADS_PER_CORE" >> $SLURM_JOB_ID.txt
+echo "Minimum memory required per CPU: $SLURM_MEM_PER_CPU" >> $SLURM_JOB_ID.txt
+echo "Requested memory per GPU: $SLURM_MEM_PER_GPU" >> $SLURM_JOB_ID.txt
+```
+
+Gaussian can consume a lot of memory, so include these additional flags in your `sbatch` file if you are running a large job:
+
+```
+#SBATCH –exclusive # The job allocation can not share nodes with other running jobs 
+#SBATCH –mem=0 # Use the maximum available memory on the node
+```
+
+You can submit this example job by running this command in your terminal:
+
+```
+sbatch water.sbatch
+```
+
```

Comment:
For calculations that involve multiple steps (such as geometry optimization, single-point calculation, frequency analysis), it is recommended to generate [checkpoints](http://www.ccl.net/cca/documents/dyoung/topics-orig/checkpoint.html) at the end of individual steps. The checkpoint of the preceding steps will be used as input for the next steps. This way, if the job crashes or is terminated due to time limit, you can resubmit the job at the suitable step to continue the calculation. For more details and guidelines, please refer to [the Gaussian documentation](https://gaussian.com/running/).